src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+
+const peopleWithParents = (people: Person[]): Person[] => {
+  return people.map(person => {
+    const mother = people.find(({ name }) => name === person.motherName);
+    const father = people.find(({ name }) => name === person.fatherName);
+
+    return ({
+      ...person,
+      mother,
+      father,
+    });
+  });
+};
```

Comment:
- You can move `peopleWithParents` function to a separate file as a utility (or helper) function.
- You don't need to pass an individual `person` as a parameter, as you need to find `mother` and `father` for all people in the list. Just go through the array of people and find their mothers and fathers.

Reply:
agree with Natali, move this fn to utils

Didn’t catch second comment about passing an individual person as a parameter.